architect_py/async_client.py

```diff
@@ -308,6 +308,62 @@ async def send_limit_order(
         )
 
         return await self.get_order(order)
+    
+    async def send_market_pro_order(
+        self,
+        *,
+        market: str,
+        dir: OrderDirection,
+        quantity: DecimalLike,
+        trigger_price: Optional[DecimalLike] = None,
```

Comment:
trigger price doesn't make sense for a market order, i think we should remove this.

let's also remove quote_id

percent_though_market may as well be a decimal

Reply:
Done.

If we're removing trigger price, do we plan to expose a stop market order as part of the interface?

Reply:
The way stop market orders work are that when the trigger price is hit it's supposed to turn into a market order. I want to avoid any kind of market orders, so it would have to be that at the trigger price it turns into market pro, but we can't determine ahead of time what a good marketable limit price will look like. So short answer is no, for now.